appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/model/OperationImpl.java

```diff
@@ -102,7 +101,16 @@ public static Operation createInstance(AnnotationModel annotation, ApiContext co
         if (requestBody != null) {
             from.setRequestBody(RequestBodyImpl.createInstance(requestBody, context));
         }
-        extractAnnotations(annotation, context, "responses", "responseCode", APIResponseImpl::createInstance, from.responses::addAPIResponse);
+        //extractAnnotations(annotation, context, "responses", "responseCode", APIResponseImpl::createInstance, from.responses::addAPIResponse);
+        List<AnnotationModel> responses = annotation.getValue("responses", List.class);
+        if (responses != null) {
+            APIResponsesImpl apiResponsesImpl = APIResponsesImpl.createInstance(annotation, context);
+// FIXME: implement!
+//            for (AnnotationModel response : responses) {
+//                from.getResponses().addAPIResponse(response.getValue("responseCode", String.class), apiResponsesImpl);
+//            }
```

Comment:
What is this? Is this required by the spec? Do we have an issue for this? Is this not required for the TCK?

Reply:
Solved now in other way, removing comment.